### ISSUE TITLE
Error executing this command "keycard init -l debug" 

### ISSUE THREAD

**Issue by valentin-dirken** [keycard-tech/keycard-cli#9]
Dear all, 
Do you know how to manage this error ? (Windows10 with keycard.exe 64bits installed)

Thanks a lot ! 

```
\Users\valen\Desktop>keycard info
[32mINFO [0m[07-08|17:30:52.129] waiting for a card                       [32mpackage[0m=status-go/cmd/keycard
[32mINFO [0m[07-08|17:30:52.157] card found                               [32mpackage[0m=status-go/cmd/keycard [32mindex[0m=0
[32mINFO [0m[07-08|17:30:52.160] info started                             [32mpackage[0m=status-go/cmd/keycard
[32mINFO [0m[07-08|17:30:52.162] select keycard applet                    [32mpackage[0m=status-go/cmd/keycard
[32mINFO [0m[07-08|17:30:52.181] select cash applet                       [32mpackage[0m=status-go/cmd/keycard
Keycard Applet:
  Installed: true
  Initialized: false
  Key Initialized: false
  InstanceUID: 0x
  SecureChannelPublicKey: 0x0458fdfaa9a447ec4faa12d
  Version: 0x
  AvailableSlots: 0x
  KeyUID: 0x
  Capabilities:
    Secure channel:true
    Key management:false
    Credentials Management:true
    NDEF:false
Cash Applet:
  Installed: true
  PublicKey: 0x0435b4d3
  Address: 0x7f3a56
  Public Data: 0x
  Version: 0x0300

C:\Users\valen\Desktop>keycard init -l debug
[32mINFO [0m[07-08|17:31:06.004] waiting for a card                       [32mpackage[0m=status-go/cmd/keycard
[32mINFO [0m[07-08|17:31:06.030] card found                               [32mpackage[0m=status-go/cmd/keycard [32mindex[0m=0
[36mDEBUG[0m[07-08|17:31:06.032] using reader                             [36mpackage[0m=status-go/cmd/keycard [36mname[0m="ACS CCID USB Reader 0"
[36mDEBUG[0m[07-08|17:31:06.036] connecting to card                       [36mpackage[0m=status-go/cmd/keycard [36mreader[0m="ACS CCID USB Reader 0"
[36mDEBUG[0m[07-08|17:31:06.323] card protocol                            [36mpackage[0m=status-go/cmd/keycard [36mT[0m=1
[32mINFO [0m[07-08|17:31:06.323] initialization started                   [32mpackage[0m=status-go/cmd/keycard
[32mINFO [0m[07-08|17:31:06.366] select keycard applet                    [32mpackage[0m=status-go/cmd/keycard
[36mDEBUG[0m[07-08|17:31:06.366] apdu command                             [36mpackage[0m=io                    [36mhex[0m="00 A4 04 00 09 A0 00 00 08 04 00 01 01 01 00"
[36mDEBUG[0m[07-08|17:31:06.392] apdu response                            [36mpackage[0m=io                    [36mhex[0m="80 41 04 58 FD FA A9 A4 47 EC 4F AA 12 D3 3D 51 B6 05 08 9D EC 7B 96 A9 B0 88 5E 6F E5 AA 83 01 8A FD 55 C5 7F E7 69 16 7B 94 B7 54 83 15 87 B4 E8 53 1D 26 64 5C 5D C3 58 10 C2 29 61 54 FC 35 F9 96 80 90 00"
[32mINFO [0m[07-08|17:31:06.393] initializing                             [32mpackage[0m=status-go/cmd/keycard
[36mDEBUG[0m[07-08|17:31:06.393] apdu command                             [36mpackage[0m=io                    [36mhex[0m="80 FE 00 00 92 41 04 6D 03 78 73 92 13 F8 30 CF 28 2E 53 9F D2 1E C2 82 6D 10 A9 0C BD 6C 82 68 6A 05 AF C8 3C 3B 73 89 6A 9E F8 81 5D 95 12 45 20 8F 0E 7D 41 7D CD D7 47 88 59 A9 06 68 AB 2C 6B A1 BC 16 82 E8 49 53 CE 92 5E C0 BF 02 F0 F3 68 0B C1 7E E0 20 AA E1 F6 9E BE 15 E3 EE 4C C8 D2 50 85 9B 8D 1B 49 D7 55 54 F9 4C 82 FA C9 81 A8 DC 83 CA 99 CF C6 20 30 D8 0B 0F 99 E2 9E E1 28 26 64 FA BB 9D 60 C6 27 D7 29 71 26 62 3B EA DE A5 B0 A7 B5 2A 0A"
[31mERROR[0m[07-08|17:31:06.571] error executing command                  [31mpackage[0m=status-go/cmd/keycard [31mcommand[0m=init [31merror[0m="scard: error(c000002b): winapi error #3221225515"

C:\Users\valen\Desktop>keycard init
[32mINFO [0m[07-08|17:35:11.165] waiting for a card                       [32mpackage[0m=status-go/cmd/keycard
[32mINFO [0m[07-08|17:35:11.190] card found                               [32mpackage[0m=status-go/cmd/keycard [32mindex[0m=0
[32mINFO [0m[07-08|17:35:11.477] initialization started                   [32mpackage[0m=status-go/cmd/keycard
[32mINFO [0m[07-08|17:35:11.519] select keycard applet                    [32mpackage[0m=status-go/cmd/keycard
[32mINFO [0m[07-08|17:35:11.546] initializing                             [32mpackage[0m=status-go/cmd/keycard
[31mERROR[0m[07-08|17:35:11.722] error executing command                  [31mpackage[0m=status-go/cmd/keycard [31mcommand[0m=init [31merror[0m="scard: error(c000002b): winapi error #3221225515"
```

**Comment by vineetpant** [comment]
Did you resolve this error, i am stuck with the same scenario while trying to run init command . I deleted the existing applet and managed to install applet but init is failing consistently

**Comment by gravityblast** [comment]
@vineetpant @valentin-dirken how did you install the applet?
I have the same problem if I install the applet with `keycard install -a /path/.cap`, but I don't have it if I install it with the shell.
So in the meantime can you try installing with:

**REMEMBER THIS IS RE-INSTALLING THE APPLET AND DELETING YOU CURRENT KEYS**
```
keycard shell -l debug < _shell-commands-examples/01-install.sh
```

**Comment by vineetpant** [comment]
@gravityblast Thank you , it works this way. Yes use case was that i was trying to reset the card . but i was not able to init after applet was installed. Is there any documentation regarding these commands?

**Comment by gravityblast** [comment]
this issue should be solved with version 0.6.0. it's not really a problem with the CLI, but it's related to the card and how many applets are installed. by default now we install all and everything should work correctly.
